buildbot/osuosl/master/config/builders.py

```diff
@@ -591,22 +591,42 @@
                         "-DMLIR_RUN_ARM_SVE_TESTS=True",
                         "-DLLVM_LIT_ARGS='-v'"])},
 
+    {'name' : "clang-arm64-windows-msvc-testsuite",
+    'tags'  : ["clang"],
+    'workernames' : ["linaro-armv8-windows-msvc-03"],
+    'builddir': "clang-arm64-windows-msvc-testsuite",
+    'factory' : ClangBuilder.getClangCMakeBuildFactory(
+                    vs="manual",
+                    checks=[],
```

Comment:
Where are you disabling the check stage 1 and 2 here? Also this doesn't seem to be a 2 stage, there's no useTwoStage.